go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/turbonomic/kubeturbo
 
-go 1.19
+go 1.20
```

Comment:
Do we want to match this version with the one specified in `build/Dockerfile.multi-archs` (i.e., `1.20.3`) ? 

Shall we also have a comment here to say that if this version is changed, we need to make sure the golang base image needs to changed as well?

Reply:
Yes, that's not allowed. if I put `1.20.3` in the go.mod, it reports:
```
[kevinw@Kevins-MacBook-Pro.local kubeturbo]$ go mod tidy
go: errors parsing go.mod:
/opt/git/kubeturbo/go.mod:3: invalid go version '1.20.3': must match format 1.23
```
I added a comment in `go.mod` to remind syncing the change to the `Dockerfile.multi-archs`

Thanks @ading1977  @amd-ibm  for helping review this change.